src/lib/ITensorMPS/src/mpo.jl

```diff
@@ -822,10 +822,11 @@ end
 function ITensors.contract(
   ::Algorithm"zipup",
   A::MPO,
-  B::MPO;
+  B::AbstractMPS;
   cutoff=1e-14,
   maxdim=maxlinkdim(A) * maxlinkdim(B),
   mindim=1,
+  truncate_kwargs=(;),
```

Comment:
```suggestion
  truncate_kwargs=(; cutoff, maxdim, mindim),
```
since I think it makes sense to use those by default in `truncate!`.

Reply:
I agree. Done.